sbnana/CAFAna/Core/FileReducer.h

```diff
@@ -53,6 +53,9 @@ namespace ana
                         const std::set<std::string>& mask,
                         const std::vector<std::string>& fnames) const;
 
+    /// Strip all information out of this record and tag it as a husk
+    void Huskify(caf::StandardRecord* sr) const;
```

Comment:
I suppose it's custom to pass this by pointer?
Since the code does not support a null `sr`, a reference would be more appropriate ([`[CF-101]`](https://sbnsoftware.github.io/sbn/codingconv/CodingConventionsList.html#CF101), but adherence to an established custom trumps it (since the custom is not very harmful).

Reply:
It is custom for Cuts and Vars at least, throughout CAFAna. It's probably a mistake, but was established in the very early days.